dashboard/app/models/studio.rb

```diff
@@ -358,7 +358,10 @@ def common_blocks(type)
 <category name="Input">
   <block type="studio_ask" />
 </category>
-<category name="Variables" custom="VARIABLE" />
+<category name="Variables" custom="VARIABLE">
```

Comment:
I'm not convinced "variables" is the best name for this; I would assume this section would give me tools for declaring and using custom variables, not getters and setters for existing variables.

That being said, I'm not entirely sure what I _would_ want to call this section